install_custom_module_name/after/Pods/Target Support Files/libPusher/libPusher.xcconfig

```diff
@@ -1,7 +1,7 @@
 CONFIGURATION_BUILD_DIR = ${PODS_CONFIGURATION_BUILD_DIR}/libPusher
 FRAMEWORK_SEARCH_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/SocketRocket"
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) COCOAPODS=1 kPTPusherClientLibraryVersion=@\"1.6.2\"
-HEADER_SEARCH_PATHS = "${PODS_ROOT}/Headers/Private" "${PODS_ROOT}/Headers/Public"
+HEADER_SEARCH_PATHS =
```

Comment:
Changing this line to https://github.com/CocoaPods/CocoaPods/pull/6966/files#diff-d598c315f60589774063d6e05dd58cf1R50 will actually get rid of this.

If you want I can add it in my own PR or it can be done as part of Swift Static Lib PR